app/views/articles/_single_story.html.erb

```diff
@@ -1,9 +1,11 @@
 <div class="crayons-story <% if featured == true %>crayons-story--featured<% end %>" data-content-user-id="<%= story.user_id %>">
   <% if featured == true %>
-  <div id="featured-story-marker" data-featured-article="articles-<%= story.id %>"></div>
-  <a href="<%= story.path %>" class="crayons-story__cover" aria-label="featured article" data-featured-article="articles-<%= story.id %>">
-    <div style="background-color: <%= story.main_image_background_hex_color %>; background-image: url(<%= cloud_cover_url(story.main_image) %>);" class="crayons-story__cover__image"></div>
-  </a>
+    <div id="featured-story-marker" data-featured-article="articles-<%= story.id %>"></div>
+  <% end %>
+  <% if featured == true || (SiteConfig.feed_style == "rich" && story.main_image.present?) %>
```

Comment:
If we intend to make this user-specific we should probably reference the value from the user view data here so when it becomes part of the user UX there is only one spot we have to update.

Reply:
Good call. This specific area is server-rendered and is only served to logged-out users, but I added a new method in `ApplicationController` where which we could call from within async controller or here and will soon serve different content if user exists and has different preferences.